tests/svg_test.py

```diff
@@ -675,14 +675,78 @@ def test_remove_processing_instructions():
     assert "xpacket" not in pico_svg.tostring()
 
 
-def test_allow_text():
-    text_svg = load_test_svg("text-before.svg")
+@pytest.mark.parametrize(
+    "svg_string, match_re, expected_passthrough",
+    [
+        # text element
+        (
+            """
+            <svg xmlns="http://www.w3.org/2000/svg"
+                width="512" height="512"
+                viewBox="0 0 30 30">
+            <text x="20" y="35">Hello</text>
+            </svg>
+            """,
+            r"Unable to convert to picosvg: BadElement: /svg\[0\]/text\[0\]",
+            "text",
+        ),
+        # text with tspan
+        (
+            """
+            <svg xmlns="http://www.w3.org/2000/svg"
+                width="512" height="512"
+                viewBox="0 0 30 30">
+            <text x="20" y="35">
+                <tspan x="0" y="20" style="font-style:normal;font-variant:normal;font-weight:normal;font-stretch:normal;font-size:10px;font-variant-ligatures:normal;font-variant-caps:small-caps;font-variant-numeric:normal;font-variant-east-asian:normal;stroke-width:1;">Hello</tspan>
+            </text>
+            </svg>
+            """,
+            r"Unable to convert to picosvg: BadElement: /svg\[0\]/text\[0\]",
+            "tspan",
+        ),
+        # tspan without text
```

Comment:
is it even valid to have tspan without a parent text element?

Reply:
Actually, you're right... It's mentioned here: https://developer.mozilla.org/en-US/docs/Web/SVG/Tutorial/Texts